rebus/chain.json

```diff
@@ -0,0 +1,225 @@
+                "address": "[2a01:4f9:6b:2e5b::9]:26656"
+            },
+            {
+                "id": "b8c42fcb311b47cdb8285b5697f661fbba5bf1a5",
+                "address": "51.68.157.129:26656"
+            },
+            {
+                "id": "e1ab57dc8fb74bc238351e063da4de1e4ed78696",
+                "address": "142.44.143.93:26656"
+            },
+            {
+                "id": "5882961bd31831bf912c0d5fe5486da100c75d6b",
+                "address": "65.109.54.110:36656"
+            },
+            {
+                "id": "011a80b75d4986be1c71267785341e172d4aa321",
+                "address": "198.244.228.17:43656"
+            },
+            {
+                "id": "f5ceadfe92dc08bb57a30977409c8b195b822dc7",
+                "address": "194.34.232.124:26656"
+            },
+            {
+                "id": "c876aaf9c3626272666bdd161155166e7534c2a0",
+                "address": "65.108.206.56:31656"
+            }
+        ]
+    },
+    "apis": {
+        "rpc": [
+            {
+                "address": "https://api.rebuschain.com:26657/",
+                "provider": "Rebuschain"
+            },
+            {
+                "address": "https://rebus.rpc.manticore.team:443/",
+                "provider": "MantiCore"
+            },
+            {
+                "address": "https://rpc.rebus.nodestake.top/",
+                "provider": "NodeStake"
+            },
+            {
+                "address": "https://rpc-1.rebus.nodes.guru",
+                "provider": "Nodes.Guru"
+            }
+        ],
+        "rest": [
+            {
+                "address": "https://api.rebuschain.com:1317/",
+                "provider": "Rebuschain"
+            },
+            {
+                "address": "https://rebus.api.manticore.team:443/",
+                "provider": "MantiCore"
+            },
+            {
+                "address": "https://api.rebus.nodestake.top/",
+                "provider": "NodeStake"
+            },
+            {
+                "address": "https://api-1.rebus.nodes.guru",
+                "provider": "Nodes.Guru"
+            }
+        ],
+        "grpc": [
+            {
+                "address": "https://rebus.grpc.manticore.team:443/",
+                "provider": "MantiCore"
+            },
+            {
+                "address": "https://grpc.rebus.nodestake.top/",
+                "provider": "NodeStake"
+            }
+        ]
+    },
+    "explorers": [
+        {
+            "kind": "explorers.guru",
+            "url": "https://rebus.explorers.guru",
+            "tx_page": "https://rebus.explorers.guru/transaction/${txHash}"
+        },
+        {
+            "kind": "ping.pub",
+            "url": "https://explorer.nodestake.top/rebus",
+            "tx_page": "https://explorer.nodestake.top/rebus/txs/${txHash}"
+        },
+        {
+            "kind": "ping.pub",
+            "url": "https://rebus.explorers.guru",
```

Comment:
this explorer object is a duplicate. Should be removed

Reply:
fixed